StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/NoXmlFileHeaderUnitTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.DocumentationRules;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// Unit tests for file header that do not follow the XML syntax.
+    /// </summary>
+    public class NoXmlFileHeaderUnitTests : DiagnosticVerifier
+    {
+        private const string SettingsFileName = "stylecop.json";
+        private const string TestSettings = @"
+{
+    ""settings"": {
+    ""documentationRules"": {
+      ""copyrightText"": ""Copyright (c) FooCorp. All rights reserved.\nLicensed under the ??? license. See LICENSE file in the project root for full license information."",
+      ""xmlHeader"": false
```

Comment:
:bulb: If you add the following property, you can test more advanced placeholders. The placeholder names and values were picked here so the tests below would not need to change.

```json
{
  "settings": {
    "documentationRules": {
      "companyName": "FooCorp",
      "copyrightText": "Copyright (c) {companyName}. All rights reserved.\nLicensed under the {licenseName} license. See {licenseFile} file in the project root for full license information.",
      "variables": {
        "licenseName": "???",
        "licenseFile": "LICENSE",
      },
      "xmlHeader": false
    }
  }
}
```